app/javascript/controllers/leaflet_viewer_controller.js

```diff
@@ -75,7 +75,7 @@ export default class LeafletViewerController extends Controller {
     if (this.availableValue && this.protocolValue) {
       await this.addPreviewOverlay();
       this.addInspection();
-    } else if (this.drawInitialBoundsValue && this.bounds) {
+    } else if ((this.drawInitialBoundsValue || !this.protocolValue) && this.bounds) {
```

Comment:
You need one or the other. Either leaflet-viewer-draw-initial-bounds-value or this line. Also I would recommend not doing this. Since you watch a multipolygon, I would change line 75 to if (this.availableValue)
and then add a line after https://github.com/geoblacklight/geoblacklight/blob/0d00b850e58e8ab92f5db97166563bb83afa3ea3/app/javascript/controllers/leaflet_viewer_controller.js#L203, and have the line be
else { return tileLayer(url, options); } which might work. I would have to double check. This definitely is going to need some tinkering.